maker-core/scopes-and-frameworks.md

```diff
@@ -24,14 +17,15 @@ Any change made to a Scope Framework must aim to improve one or more of the foll
 - Simplicity of protocol
 
 ## Governance
-Each Scope has an Advisory Council. This Council consists of members who must fulfill certain requirements and have been approved by Maker Governance. The Council carries out advisory work related to improving the content of the Scope Framework.
+Each Scope has a Scope Advisory Council. This Council consists of members who must fulfill certain requirements and have been approved by Maker Governance. The Council carries out advisory work related to improving the content of the Scope Artifact.
 
-The exception to this is the Arbitration Scope, which does not have an Advisory Council.
+The exception to this is the Governance Scope, which does not have a Scope Advisory Council.
```

Comment:
Can we double check that this is still the case? 2.1 of the Atlas seems to suggest otherwise